miso-web/src/main/webapp/scripts/sequencer_partition_container_ajax.js

```diff
@@ -282,7 +282,7 @@ Container.ui = {
       {
         'updateElement': 'containerdiv',
         'doOnSuccess': function(json) {
-            jQuery('#containerdiv').find('input[type="radio"]').first().click();
+          jQuery('#containerdiv').find('input.selectMe').click();
```

Comment:
The partitions are fetched by a separate AJAX call which is triggered by the click, and the string concatenation and session attribute setting in the ContainerControllerHelperService is complex enough that refactoring it would not be straightforward (for me). I'd prefer to fix the behaviour now and refactor the functions if/when we work out the ControllerHelperServices.